### PR TITLE
Content page

### PR DESCRIPTION
**Pull request by chris-lawton**
Some general Streamfield fixes now the content page BE has been done. 

I've added a `sf-spacing` class so the spacing between streamfields should be consistent.

### CODE REVIEW

File: wagtailio/project_styleguide/templates/patterns/components/streamfields/rich_text_block/rich_text_block.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {% load wagtailcore_tags %}
 
-<div>
-    {{ value.rich_text }}
+<div class="grid">
+    <div class="rich-text rich-text--sf">
+        {{ value.rich_text }}
```
Comment (stevedya) [inline wagtailio/project_styleguide/templates/patterns/components/streamfields/rich_text_block/rich_text_block.html:5]: Question: Do we need the `|richtext` filter on here?

File: wagtailio/static/sass/abstracts/_mixins.scss [hunk 2]
```diff
@@ -28,3 +28,12 @@
     white-space: nowrap;
     border: 0;
 }
+
+// Streamfield spacing
+@mixin sf-spacing() {
```
Comment (stevedya) [inline wagtailio/static/sass/abstracts/_mixins.scss:33]: **Praise:** Noice! Definitely needed this

Comment (stevedya) [reply wagtailio/static/sass/abstracts/_mixins.scss:33]: We could use this as a block spacer throughout other parts of the site as well (not just for streamfields). Perhaps in the future we can add params for specifying top or bottom spacing or both for different scenarios. Not required as part of this work though 😅

**Review by stevedya — APPROVED** [review]
Looking great! Love the addition of the sf-spacing mixin

File: wagtailio/project_styleguide/templates/patterns/components/streamfields/rich_text_block/rich_text_block.html (hunk 1 above)
Comment (chris-lawton) [reply wagtailio/project_styleguide/templates/patterns/components/streamfields/rich_text_block/rich_text_block.html:5]: I think we do! : D

File: wagtailio/static/sass/abstracts/_mixins.scss (hunk 2 above)
Comment (chris-lawton) [reply wagtailio/static/sass/abstracts/_mixins.scss:33]: That sounds useful. The streamfields had pretty consistent spacing in the design but no reason why we couldn't rename and adopt this for other components - would be nice to have it all controlled from one source : )

**Pull request merged**